scripts/generate-config.sh

```diff
@@ -39,12 +39,22 @@ compare_extensions=(
   yaml
   go
   proto_descriptor
+  html
 )
 
+exit_with=0
+echo "Generating config against istio $(cd ${istio_src} && git describe --contains --all HEAD)"
 for i in "${compare_extensions[@]}" ; do
   for file in "${target_dir}/config/"*.${i} ; do
-    diff -b "$file" "config/${file##*/}"
+    changed=$(diff -b "$file" "${root_dir}/config/${file##*/}")
+    if [[ $? != 0 ]];
```

Comment:
How about instead of adding an unused variable to suppress output we direct all output to `/dev/null` or using the `--brief` option which outputs when files differ (good to check it is also supported in OSX)? We would still need to keep this `if`, but directly invoking the tool should be fine `if diff --brief "${file}" "${root_dir}/config/${file##*/}"; then`